apps/comps/app/api/nonce/route.ts

```diff
@@ -0,0 +1,27 @@
+import { serialize } from "cookie";
+import { NextApiRequest } from "next";
+import { NextResponse } from "next/server";
+import { generateSiweNonce } from "viem/siwe";
+
+export async function GET(req: NextApiRequest) {
+  try {
+    const nonce = generateSiweNonce();
+
+    const cookie = serialize("session", nonce, {
+      httpOnly: true,
+      secure: true,
+      maxAge: 60 * 60 * 24 * 7, // One week
+      path: "/",
+    });
+
```

Comment:
why not `import { cookies } from 'next/headers'` instead of a new dep?
https://nextjs.org/docs/app/api-reference/functions/cookies